source/tasks/Utils/environment.ts

```diff
@@ -43,6 +44,7 @@ export const getVstsEnvironmentVariables= () : VstsEnvironmentVariables =>{
         teamCollectionUri: process.env["SYSTEM_TEAMFOUNDATIONCOLLECTIONURI"],
         defaultWorkingDirectory: process.env["SYSTEM_DEFAULTWORKINGDIRECTORY"],
         buildRepositoryProvider: process.env["BUILD_REPOSITORY_PROVIDER"],
+        buildRepositoryUri: process.env["BUILD_REPOSITORY_URI"]
```

Comment:
I couldn't see where the `buildRepositoryUri` is used. Should it stay in case you want to use it later, or should it be removed?

Reply:
I was going to use the value but decided not to. But I didn't see any harm in leaving this here for future reference.